use-case-guides/workday-request-time-off/README.md

```diff
@@ -3,87 +3,53 @@ design_pattern_id: 2
+3. Do you specify time off in days or hours for these plans?
+    * This determines the what slot type will be caputured for time off requests, which affects the validation provided to employees
+4. What are the required fields for managers to review time off requests (example: comments, business justification). Which are optional?
+    * This affects what questions will be asked to employees by your copilot when they submit a time off request
+5. Can your employees request time off requests only for themselves, or also for someone else?
    * Note: this guide only allows employees to request time off for themselves. See API research below to understand how you can customize API research to allow time off requests for other employees
 
+
 # For Workday Administrators
 
-This document provides a step-by-step guide designed for Workday Administrators and HR Systems Engineers, detailing the procedures for creating credentials, assigning roles, and configuring workday effectively. It serves as a practical manual to ensure accurate setup and management of technical resources.
+**Level of Access Requested**:
+1. Configure a [Workday connector](https://developer.moveworks.com/creator-studio/resources/authentication-guide?id=workday) with your Workday administrator
+2. Provide access to the `Worker Data` and `Time Off` Business Objects. Elaborated in [Domain Permissions](#domain-permissions-required) section
```

Comment:
Either fix the markdown renderer code to add `id` tags to heading divs, or remove the link and mention "below"

Reply:
addressed in new text